README.md

```diff
@@ -5,12 +5,104 @@ Copyright (C) 2017-2023 The Open Library Foundation
 This software is distributed under the terms of the Apache License,
 Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
 
+## Table of contents
+
+- [Introduction](#introduction)
+- [Permissions](#permissions)
+- [API](#api)
+  - [Configuration API](#configuration-api)
+    - [Updating a configuration](#updating-a-configuration)
+- [Additional information](#additional-information)
+  - [Other documentation](#other-documentation)
+  - [Issue tracker](#issue-tracker)
+  - [Quick start](#quick-start)
+  - [API documentation](#api-documentation)
+  - [Code analysis](#code-analysis)
+  - [Download and configuration](#download-and-configuration)
+
 ## Introduction
 
 The mod-audit module provides API to access and modify audit data.
 
 ## Permissions
 
+## API
+
+### Configuration API
```

Comment:
Should be mentioned that user should have special permissions to view each group settings, and special permission for each setting to update it.